addons/website/static/src/components/dialog/edit_menu.js

```diff
@@ -198,7 +198,9 @@ export class EditMenuDialog extends Component {
         const parent = this.map.get(parentId);
         parent.children = parent.children.filter(menu => menu.fields['id'] !== id);
         this.map.delete(id);
-        this.toDelete.push(id);
+        if (parseInt(id)) {
```

Comment:
Could've deserved a comment, it's really not obvious that this is filtering out the menu which were deleted after having being created in the current menu edition (and so never actually existed in the DB).